backend/python/app/models/entity_mg.py

```diff
@@ -0,0 +1,18 @@
+from mongoengine import *
+from .enum import enum
+
+
+class Entity(Document):
+    string_field = StringField(max_length=200, required=True)
+    int_field = IntField(required=True)
+    enum_field = StringField(
+        max_length=200, required=True, choices=["A", "B", "C", "D"]
+    )
+    string_array_field = ListField(StringField(max_length=200))
+    bool_field = BooleanField()
+
```

Comment:
Can you add this additional line into the model? Just so the collection name is properly plural-ized

```py
meta = {"collection": "entities"}
```